packages/palette/src/elements/BaseTabs/BaseTab.story.tsx

```diff
@@ -0,0 +1,182 @@
+import { action } from "@storybook/addon-actions"
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { BaseTab, BaseTabs } from "."
+import { ChevronIcon } from "../../svgs"
+import { Box } from "../Box"
+import { Clickable } from "../Clickable"
+
+// Fake `RouterLink`
+const RouterLink: React.FC<{ to: string }> = ({ to, children, ...rest }) => {
+  return (
+    <a href={to} {...rest}>
+      {children}
+    </a>
+  )
+}
+
+storiesOf("Components/BaseTabs", module)
+  .add("Default", () => {
+    return (
+      <BaseTabs m={3}>
+        <BaseTab>First</BaseTab>
+        <BaseTab>Second</BaseTab>
+        <BaseTab active>Active</BaseTab>
+        <BaseTab>Last</BaseTab>
+      </BaseTabs>
+    )
+  })
+  .add('as="a"', () => {
+    return (
+      <BaseTabs m={3}>
+        <BaseTab as="a" href="#overview">
+          Overview
+        </BaseTab>
+
+        <BaseTab as="a" href="#cv">
+          CV
+        </BaseTab>
+
+        <BaseTab as="a" href="#shows" active>
+          Shows
+        </BaseTab>
+
+        <BaseTab as="a" href="#auction-results">
+          Auction Results
+        </BaseTab>
+
+        <BaseTab as="a" href="#articles">
+          Articles
+        </BaseTab>
+
+        <BaseTab as="a" href="#related-artists">
+          Related Artists
+        </BaseTab>
+      </BaseTabs>
+    )
+  })
+  .add("as={Clickable}", () => {
+    return (
+      <BaseTabs m={3}>
+        <BaseTab as={Clickable} onClick={action("onClick")}>
+          Overview
+        </BaseTab>
+
+        <BaseTab as={Clickable} onClick={action("onClick")}>
+          CV
+        </BaseTab>
+
+        <BaseTab as={Clickable} onClick={action("onClick")} active>
+          Shows
+        </BaseTab>
+
+        <BaseTab as={Clickable} onClick={action("onClick")}>
+          Auction Results
+        </BaseTab>
+
+        <BaseTab as={Clickable} onClick={action("onClick")}>
+          Articles
+        </BaseTab>
+
+        <BaseTab as={Clickable} onClick={action("onClick")}>
+          Related Artists
+        </BaseTab>
+      </BaseTabs>
+    )
+  })
+  .add("as={RouterLink}", () => {
+    return (
+      <BaseTabs m={3}>
+        <BaseTab as={RouterLink} to="#example" activeClassName="active" garbage>
+          Example
+        </BaseTab>
+      </BaseTabs>
+    )
+  })
+  .add("Overflowing", () => {
+    return (
+      <BaseTabs m={3}>
+        {[
+          "Artworks (580656)",
+          "Artists (8)",
+          "Collections (7)",
+          "Artist Series (13)",
+          "Galleries (4)",
+          "Shows (3667)",
+          "Categories (21)",
+          "Articles (656)",
+          "Auctions (23)",
+          "More (46)",
+        ].map((label, i) => {
+          return (
+            <BaseTab key={label} active={i === 1}>
+              {label}
+            </BaseTab>
+          )
+        })}
+      </BaseTabs>
+    )
+  })
+  .add("Fill", () => {
```

Comment:
What does `fill` mean?

Reply:
This pattern was mentioned in the Figma specs:

![](http://static.damonzucconi.com/_capture/F6eJ7Fmv2I6c.png)

Reply:
_finally_